config/build.ts

```diff
@@ -430,14 +436,16 @@ function checkSnippetsForUniqueIDs(processedSnippets) {
     }
 }
 
-async function generatePlaylists(dir, processedSnippets: Dictionary<SnippetProcessedData>) {
+async function generatePlaylists(dir, processedSnippets: Dictionary<SnippetProcessedData>, isPrivateSample?) {
     banner('Generating playlists');
 
-    /* Creating playlists directory */
-    status.add(`Creating \'${dir}\' folder`);
-    await rmRf(dir);
-    await mkDir(dir);
-    status.complete(true /*success*/, `Creating \'${dir}\' folder`);
+    if (!isPrivateSample) {
+        /* Creating playlists directory */
+        status.add(`Creating \'${dir}\' folder`);
+        await rmRf(dir);
+        await mkDir(dir);
+        status.complete(true /*success*/, `Creating \'${dir}\' folder`);
+    }
 
     const groups = groupBy(
```

Comment:
Sorry the code review does not seem to let me comment on existing lines. Below lines are causing the issue I believe.
        const groupNumberRegex = /^(\d+\s)?(\w.*)$/;

        items.forEach(item => {
            item.group = item.group.replace(groupNumberRegex, '$2');

            // Also remove "order", it's no longer needed (the snippets themselves are already in an ordered array in the YAML file)
            delete item.order;
        });